CONTRIBUTING.md

```diff
@@ -312,12 +318,12 @@ the "standard" release naming conventions, with `-rcX` at the end (`vX.X.X-rcX`)
 have distinct names from the tags/release names.)
 
 1. Start from the RC branch (e.g. `RC0/v0.34.0`).
-2. Create the new tag, specifying a name and a tag "message":  
-   `git tag -a v0.34.0-rc0 -m "Release Candidate v0.34.0-rc0` 
-3. Push the tag back up to origin:  
```

Comment:
These spaces at the end of the line are deliberate - to force a newline before the git commands. Can you add them back, please?

Reply:
sorry this was done automatically, will revert.